dev/ory/oathkeeper_rules.yaml

```diff
@@ -77,11 +95,11 @@
       config:
         claims: '{"sub": "{{ print .Subject }}"}'
 
-- id: galoy-backend
+- id: galoy-backend-other-routes
```

Comment:
name is not great imo. is there something more descriptive we can have?

it's not great because we don't know from the name what route the "non other" represents